client/jetpack-connect/style.scss

```diff
@@ -553,3 +553,25 @@
 		}
 	}
 }
+
+@media ( min-height: 921px ) {
```

Comment:
Noting we should clean this repetition up after the test is done. Perhaps it might be worth to add a `TODO` comment here about it.

Reply:
great idea